### ISSUE TITLE
Request: Tabbed Interface

### ISSUE THREAD

**Issue by bladewolf55** [mike-ward/Markdown-Edit#210]
I didn't see this already in the Issues. A tabbed interface would be nice, for people who work on multiple documents (which I think is most of us). It's especially handy for copy/paste between docs. Double-click-opening a document into a new tab would likewise be welcome.

**Comment by mike-ward** [comment]
Unfortunately, you're talking to a guy that hates tabbed editors. I know it's a must for some and that's OK. There are lots of choices out there.

**Comment by bladewolf55** [comment]
Hah! Sadly, having evaluated most of the polished choices (over half a dozen), yours is overall the best. The only alternative I liked was MarkdownPad, which has been abandoned.

If I get time (doubtful), I may look into adding tabs as an optional feature and submit a pull request. Even if you don't accept, I'm sure I'd learn a lot. I could also submit a couple of other small nice-to-haves that probably wouldn't matter to most people, like auto-opening documents to the end (good for journals).

By the way, I love snippets. Very useful.

**Comment by mike-ward** [comment]
Have you looked at Markdown Monster?

**Comment by bladewolf55** [comment]
Yes. It may get there, and I respect Rick Strahl's work, but I found too
many problems in my first usage. I've got a backlog task to write a blog
entry comparing Markdown editors, and a list of my "ideal" features.

Thanks for getting back to me, and the work you're doing. (Oh, and another
feature of yours I like: drag/drop images and insert local file path.)


--
Charles L Flatt
513-535-0764 (m)   |   www.softwaremeadows.com

"There are two seasons in Scotland: June and Winter." --Billy Connolly

**Comment by Li-An** [comment]
Write! (paying) has tabs but incomplete Markdown.

**Comment by Icecream-Burglar** [comment]
Would you accept a tabbed interface if it was a selected option and defaulted to off? @mike-ward

**Comment by bladewolf55** [comment]
I think that would be an ideal implementation. Thanks for reconsidering the
feature!


--
Charles L Flatt
513-535-0764 (m)

"It is better to know less than to know so much that ain't so." --Josh
Billings

**Comment by bladewolf55** [comment]
Sorry, @Icecream-Burglar , I thought it was @mike-ward writing. That said, I think an optional tabbed interface would be welcomed by many users.
